### PR TITLE
Preparing next release 1.3.10 Second PR

### PR DESCRIPTION
**Pull request by juanky201271**
superseded by #654

### CODE REVIEW

File: app/AppState/enums/CommandEnum.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  value_to_address = 'value_to_address',
+  sends_to_address = 'sends_to_address',
+  memobytes_to_address = 'memobytes_to_address',
+
+  // new commands, some are comming...
+  send = 'send',
+  shield = 'shield',
+  // this works for both: send & shield
+  confirm = 'confirm',
+}
```
Comment (juanky201271) [inline app/AppState/enums/CommandEnum.ts:35]: those are the new commands... well, the only new command is `confirm`.

File: app/translations/en.json [hunk 2]
```diff
@@ -82,7 +83,9 @@
     "alert-newwallet-title": "CREATE A NEW WALLET",
     "alert-newwallet-body": "Can you confirm that you really want to create a new wallet and lose access to the current wallet.",
     "serverfirsttry": "Trying to perform the action with a different server. The current server may be failing.",
-    "serversecondtry": "It doesn't seem to be any problem with the server. Someting else is happening."
+    "serversecondtry": "It doesn't seem to be a problem with the server. Someting else is happening.",
```
Comment (juanky201271) [inline app/translations/en.json:86]: @Oscar-Pepper I changed it here.

File: app/translations/en.json [hunk 3]
```diff
@@ -387,7 +429,7 @@
     "nofetchprice": "Fetch Price in USD",
     "sendall": "Send All",
     "sendall-message": "Your are about to send the entire amount in your wallet",
-    "includeua": "Include your UA:",
+    "includeua": "Include your return address:",
```
Comment (juanky201271) [inline app/translations/en.json:432]: @Oscar-Pepper I changed it here.

File: app/utils/Utils.ts [hunk 4]
```diff
@@ -188,4 +193,76 @@ export default class Utils {
...
+        const amount = parseInt((Utils.parseStringLocaleToNumberFloat(to.amount) * 10 ** 8).toFixed(0), 10);
+
+        const myAddress: AddressClass[] = addresses.filter((a: AddressClass) => a.address === to.to);
+        sendToSelf = myAddress.length >= 1;
+
+        donationAddress = to.to === (await Utils.getDonationAddress(server.chain_name));
+
+        if (memo === '') {
+          return [{ address: to.to, amount } as SendJsonToTypeType];
+        } else if (Buffer.byteLength(memo, 'utf8') <= GlobalConst.memoMaxLength) {
```
Comment (juanky201271) [inline app/utils/Utils.ts:217]: @Oscar-Pepper changed to 511.

File: app/AppState/enums/CommandEnum.ts (hunk 1 above)
Comment (Oscar-Pepper) [reply app/AppState/enums/CommandEnum.ts:35]: ok great thanks

**Pull request closed without merging**